features/apimgt/org.wso2.carbon.apimgt.core.feature/src/main/resources/sql/mssql.sql

```diff
@@ -2485,6 +2485,25 @@ CREATE TABLE AM_SYSTEM_CONFIGS
   PRIMARY KEY (ORGANIZATION,CONFIG_TYPE)
 );
 
+IF NOT  EXISTS (SELECT * FROM SYS.OBJECTS WHERE OBJECT_ID = OBJECT_ID(N'[DBO].[AM_API_THROTTLE_LIMIT]') AND TYPE IN (N'U'))
+
+CREATE TABLE AM_API_THROTTLE_LIMIT (
+    API_UUID VARCHAR(256),
```

Comment:
reduce the size to 64

Reply:
can't change as MSSQL has a constraint where size of the foreign key should be as same as the referenced key.